Godnoken/DOM-I
# DOM 1 - LambdaSchool

​
## What does this PR do?
DOM 1  I Project MVP

## Description of tasks completed
- Implemented DOM selectors pointing to HTML elements
- Changed DOM selectors to using the JSON data provided
- Implemented changes and addons to the navigation
​
## How can this be tested?
Clone the project.
Open up the project files in your favourite code editor.
Compare index.html and original.html code.
Run index.html to see index.js apply DOM to the content of the page.
​
## Relevant screenshot (if applicable)
https://i.gyazo.com/afcfbd59181c9dfb34234a04aa9acffb.png
